tool/teleport/common/teleport.go

```diff
@@ -476,11 +476,13 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	integrationConfDeployServiceCmd.Flag("role", "The AWS Role used by the AWS OIDC Integration.").Required().StringVar(&ccf.IntegrationConfDeployServiceIAMArguments.Role)
 	integrationConfDeployServiceCmd.Flag("task-role", "The AWS Role to be used by the deployed service.").Required().StringVar(&ccf.IntegrationConfDeployServiceIAMArguments.TaskRole)
 	integrationConfDeployServiceCmd.Flag("aws-account-id", "The AWS account ID.").StringVar(&ccf.IntegrationConfDeployServiceIAMArguments.AccountID)
+	integrationConfDeployServiceCmd.Flag("confirm", "Do not prompt user and auto-confirm all actions.").BoolVar(&ccf.IntegrationConfDeployServiceIAMArguments.AutoConfirm)
```

Comment:
`Do not prompt user and auto-confirm all actions.` This reads relevant for a developer but not to the user executing the command. 

It should just say `Apply changes without confirmation.`

Reply:
thanks, I like that better. I copied this description from another command, so I'll update all of them with this more useful description.

Reply:
fba71a82437d0b3576465c7295008622bc8a1263